### DISCUSSION TITLE
Running Ontop from VSCode: Where should I put the DB2 driver?

### DISCUSSION THREAD

**Discussion by marcelomachado** [ontop/ontop#751] [Q&A]
I'm trying to run the `endpoint` command from vscode. I created a command in the launcher file containing the mapping data, ontology and properties file:
```
{
             "type": "java",
             "name": "Ontop",
             "request": "launch",
             "mainClass": "it.unibz.inf.ontop.cli.Ontop",
             "projectName": "ontop-cli",
             "args": ["endpoint",
             "-m", "...",
             "-t", "...",
             "-p", "...",
             "--dev"
             ]
},
```
 In my properties file I have the following information: `jdbc.driver=com.ibm.db2.jcc.DB2Driver`.

Where should I put the `jar file` since I am having the following error when trying to run: `Cannot load the driver: com.ibm.db2.jcc.DB2Driver`

**Comment by bcogrel** [comment]
If you use Maven, you can add it as dependency for your project:

```xml
        <dependency>
            <groupId>com.ibm.db2</groupId>
            <artifactId>jcc</artifactId>
            <version>11.5.7.0</version>
        </dependency>
```

Otherwise, you can also add the jar manually to the classpath. I am not experienced with Java projects in Vscode (I personally use IntelliJ), but the following page explains how to proceed: https://code.visualstudio.com/docs/java/java-project .
